apps/design-system/src/pages/view-preview/project-settings-wrapper.tsx

```diff
@@ -1,13 +1,31 @@
-import { PropsWithChildren } from 'react'
+import { FC, PropsWithChildren } from 'react'
 import { Route } from 'react-router-dom'
 
+import { ProjectSettingsPage } from '@harnessio/ui/views'
+
 import RootViewWrapper from './root-view-wrapper'
 
-export const ProjectSettingsWrapper: React.FC<PropsWithChildren<unknown>> = ({ children }) => {
+const Layout = () => {
+  return (
+    <div className="bg-background-1 sticky top-[55px] z-40">
```

Comment:
```suggestion
    <div className="bg-background-1 sticky top-14 z-40">
```
`text-14` equals `56px`, so a 1px shift shouldn't cause any issues. However, this adjustment allows us to take advantage of Tailwind's grid system